src/megachatapi.h

```diff
@@ -138,6 +149,42 @@ class MegaChatSession
      */
     virtual bool hasVideo() const;
 
+    /**
+     * @brief Returns the termination code for this session
```

Comment:
Maybe, based on all those termcodes, it's going to be hard for app's devs to understand which cases the user has left the call vs. cases where there user has drop :thinking:

Reply:
Simply indicate in the explanation of the codes whether it means the user has left, or the user has dropped.